layouts/_default/baseof.html

```diff
@@ -2,6 +2,9 @@
 <html lang="{{ .Site.Language.Lang }}">
 
   <head>
+    {{ if and .Site.Params.googleTagManager .Site.Params.googleTagManager.id }}
+    {{- partial "analytics/googletagmanager" . -}}
+    {{ end }}
```

Comment:
I just need to ask you to move these part to the end of the `body` tag, next to the other analytics templates. It doesn't seem like this code really needs to be at the `head` section.

Everything else is fine. :)

Reply:
I was thinking to do so, but google asks to do it differently:
``` Copy and paste this code as the first item into the <head> of every webpage you want to measure. ```

Should we ignore it?

Reply:
Sorry for taking too long.

I don't see why moving it would make it not work. Do you mind trying to move the code and testing it?

Reply:
It works, even when placed lower on the page, however, it's highly recommended to have it in```< head>``` for the following reasons:
 - load as early as possible to make sure that no users are missed
 - A/B testing will not work properly if placed too low

So it's important to keep it in ```<head>```, but we can move it below significant ```<meta>``` tags.

Here are the links with more info:
https://www.bounteous.com/insights/2016/11/22/where-should-google-tag-manager-snippet-be-placed/
https://stackoverflow.com/questions/46064817/how-high-in-the-head-section-can-you-safely-put-the-google-tag-manager-code


do you still feel that we better move it to the bottom of the page?

Reply:
Hmm... these arguments didn't convince me. 
I prefer to keep things consistent, so we should place it at the end of `body`.

Reply:
It's changed. Let's merge it :-)